packages/remix-server-runtime/__tests__/server-test.ts

```diff
@@ -164,11 +164,11 @@ describe("shared server runtime", () => {
           loader() {
             let headers = new Headers({ "x-test": "yes" });
             let headersProxy = new Proxy(headers, {
-              get(target, prop, receiver) {
+              get(target, prop) {
                 if (prop === "set") {
                   throw new TypeError("immutable");
                 }
-                return Reflect.get(target, prop, receiver);
+                return target[prop].bind(target);
```

Comment:
Had to change the way these tests were written to avoid the following error in `6.21.2`:

```
TypeError: Cannot read private member #headersList from an object whose class did not declare it
```